### PR TITLE
homework 14(Library)

### PR DESCRIPTION
**Pull request by Pokosteler**
homework 14(Library)

### CODE REVIEW

File: src/lesson14/library/Library.java [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+            book.setAmount(Integer.valueOf(dataOfBook.get(7)));
+            books.add(book);
+        }
+        return books;
+    }
+
+    public List<Customer> createCustomer(List<List<String>> listOfCustomers) {
+        List<Customer> customers = new ArrayList<>();
+        for (List<String> dataOfCustomer : listOfCustomers) {
+            Customer customer = new Customer();
```
Comment (TheFallen058) [inline src/lesson14/library/Library.java]: Please check your classes. Customer doesn't have default constructor so only constructor with string arg is available. Code can't be compiled

File: src/lesson14/library/Library.java [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+        return allCustomers;
+    }
+
+    public void setAllCustomers(List<Customer> allCustomers) {
+        this.allCustomers = allCustomers;
+    }
+
+    public void takeBookOutLibraryOrNot(Customer customer, Book book) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Do you want to take it out library or to the reading room?");
```
Comment (TheFallen058) [inline src/lesson14/library/Library.java]: not clear what to enter as response

**Review by TheFallen058 — CHANGES_REQUESTED** [review]
In general good work
Only sad thing that I can't compile the code without changes. Please fix

File: src/lesson14/library/Library.java (hunk 1 above)
Comment (Pokosteler) [reply src/lesson14/library/Library.java]: fixed

File: src/lesson14/library/Library.java (hunk 2 above)
Comment (Pokosteler) [reply src/lesson14/library/Library.java]: 0 - out library, 1 - to the reading room. I added an explanation to the code

**Review by TheFallen058 — APPROVED** [review]
well done

**Pull request merged**